packages/bus-http/lib/ActorHttp.ts

```diff
@@ -2,8 +2,9 @@ import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@co
 import { Actor } from '@comunica/core';
 import { ReadableWebToNodeStream } from 'readable-web-to-node-stream';
 
+global.ReadableStream = global.ReadableStream || require('web-streams-ponyfill').ReadableStream;
```

Comment:
Setting globals in JS is not considered good practise, as it can pollute the global space when working with multiple libraries together.

Can we remove this line?

Reply:
Sadly we cant because `web-streams-node` expects `ReadableStream` to be availlable globally. This global expose was done by `web-streams-node` (it's the only line of  `web-streams-node` that was useful for Comunica).

Reply:
Ah, you're right. It's not nice, but we should indeed keep the existing behaviour.

Can you put a `// TODO:` above it, and say that this can be removed once we make Node.js 18 the minimal version?

Reply:
Done.